src/sentry/api/serializers/models/event.py

```diff
@@ -269,11 +267,8 @@ def serialize(self, obj, attrs, user):
             'metadata': obj.get_event_metadata(),
             'tags': tags,
             'platform': obj.platform,
-            'dateCreated': obj.datetime,
```

Comment:
I noticed a few dependencies on dateCreated 

>src/sentry/static/sentry/app/views/organizationEventsV2/eventModalContent.jsx:          date={getDynamicText({value: event.dateCreated, fixed: 'Dummy timestamp'})}
>src/sentry/static/sentry/app/views/organizationEventsV2/modalLineGraph.jsx:  const eventTime = +new Date(currentEvent.dateCreated);
>src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx:    const dateCreated = moment(evt.dateCreated);

Are these going to be ok if you remove the field? I suspect there is no acceptance test that go through transaction event.

Reply:
Good finds. The modal graph and content component do need to be updated. The GroupDetailsToolbar should be fine as transaction events won't have groups.